CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All versions prior to 0.9.0 are untracked.
 
 ### Fixed
 
+* Avoid pydantic's instantiation issues with `TransparencyLogEntry` when `InclusionPromise` is not present.
+
```

Comment:
I don't think this is really related to pydantic (although I appreciate the neater handling in `_to_rekor()`). Isn't the issue that our code just won't work without an inclusion promise currently?

Reply:
removed mention of pydantic.